src/main/kotlin/click/seichi/observerutils/commands/Command.kt

```diff
@@ -57,12 +57,12 @@ object Commands {
                     return@execution Ok(Effect.MessageEffect("${ChatColor.RED}保護がありません。"))
                 }
                 val duplicatedRegions =
-                    if (regions.size >= 2) "(${regions.size}): ${regions.joinToString { it.id }}" else "-"
+                    if (regions.size >= 2) "(${regions.size}): @@${regions.joinToString { it.id }}@@" else "-"
```

Comment:
```suggestion
                    if (regions.size == 1) "-" else "(${regions.size}): ${regions.joinToString { "@@${it.id}@@" }}"
```
もっと言えばこっちのほうがシンプルですね。自分のコードですけど...。